recipe/meta.yaml

```diff
@@ -1,55 +1,39 @@
-{% set version = "4.4.12" %}
+{% set version = "7.1.15" %}
 
 package:
   name: pysnmp
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pysnmp/pysnmp-{{ version }}.tar.gz
-  sha256: 0c3dbef2f958caca96071fe5c19de43e9c1b0484ab02a0cf08b190bcee768ba9
-  patches:
-    - setuptools_version.patch
+  sha256: f624f2fab9503431f6514ad388b32aa657b36da59d2bc9b6059786db7d9f6257
 
 build:
-  number: 1
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
-  build:
-    - patch     # [not win]
-    - m2-patch  # [win]
   host:
     - python
+    - poetry-core >=1.0.0
     - pip
   run:
     - python
-    - pyasn1 >=0.2.3
-    - pysmi
-    - pycryptodomex
-    - twisted
+    - pyasn1 >=0.4.8,!=0.5.0
+    - pysmi >=1.3.0,<2.0.0
```

Comment:
pysmi is not listed as a required dependency, but it was set before, and is also set on the conda-forge recipe. So ok to keep.

Reply:
Ok thanks!